src/Hydrator/EloquentHydrator.php

```diff
@@ -115,6 +115,32 @@ public function __construct(HttpServiceInterface $service)
     {
         $this->service = $service;
     }
+    
+    /**
+     * @param StandardObjectInterface $attributes
+     * @param $record
+     * @return void
+     */
+    protected function defaultHydrateAttributes(StandardObjectInterface $attributes, $record)
+    {
+        $hydratorAttributes = $this->attributes;
+        if(empty($hydratorAttributes))
```

Comment:
this needs to use `is_null` instead of `empty`. I.e. an empty array can mean "hydrate no attributes" and a `null` value can mean "hydrate fillable attributes"